static/app/components/autoComplete.tsx

```diff
@@ -231,7 +254,7 @@ class AutoComplete<T extends Item> extends React.Component<Props<T>, State<T>> {
       if (hasHighlightedItem && (canSelectWithEnter || canSelectWithTab)) {
         const item = this.items.get(this.state.highlightedIndex);
 
-        if (!item.disabled) {
+        if (item && !item.disabled) {
```

Comment:
`item && !item.disabled ` can be  `!item?.disabled `

Reply:
It could if I didn’t want to also tell typescript that item is set for the enclosed function call :)